### PR TITLE
CI: improve golangci job

### PR DESCRIPTION
**Pull request by bart0sh**
- used the same go version as for the project build
- used verbose output
- fixed gofmt check failures

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -26,7 +29,10 @@ jobs:
         uses: golangci/golangci-lint-action@v2
         with:
           version: v1.42.0
```
Comment (mythi) [inline .github/workflows/ci.yaml:31]: @bart0sh set this to `v1.43.0`

Comment (bart0sh) [reply .github/workflows/ci.yaml]: I will. However, I have 1.42 installed locally and the issue is reproducible with it.

Comment (bart0sh) [reply .github/workflows/ci.yaml]: @mythi it helped to reproduce the issue. Thank you!

Comment (bart0sh) [reply .github/workflows/ci.yaml]: But still it's strange. I've just double-checked. I have 1.42.0 installed.

Comment (bart0sh) [reply .github/workflows/ci.yaml]: I suspect that golangci-lint cache is working incorrectly. It got invalidated when golangci-lint version changed, but it didn't work correctly with 1.42.0:
```
Cache Size: ~1 MB (1167060 B)
  /usr/bin/tar --use-compress-program zstd -d -xf /home/runner/work/_temp/33020325-899f-496c-97fa-e60da6f9d24c/cache.tzst -P -C /home/runner/work/intel-device-plugins-for-kubernetes/intel-device-plugins-for-kubernetes
  Cache restored successfully
  Restored cache for golangci-lint from key 'golangci-lint.cache-2706-eba1025cc7113b9b4eb9e033f9837601b5b99ec4' in 576ms
```

Any ideas how to disable this caching?

Comment (bart0sh) [reply .github/workflows/ci.yaml]: There is a way to clean cache by running 'golangci-lint cache clean' but this command is not supported by the golangci/golangci-lint-action@v2

I upgraded to 1.43.0 and fixed gofmt warnings. This is better than nothing.

**Review by mythi — APPROVED** [review]

**Comment by bart0sh** [comment]
@rojkov @kad PTAL

**Review by rojkov — APPROVED** [review]

**Pull request merged**